snippets/product-variant-picker.liquid

```diff
@@ -0,0 +1,98 @@
+{% comment %}
+    Renders product variant-picker
+
+    Accepts:
+    - product: {Object} product object.
+    - block: {Object} passing the block information.
+    - update_url: {Boolean} whether or not to update url when changing variants. If false, the url isn't updated. Default: true (optional).
+
+    Usage:
+    {% render 'product-variant-picker', block: block, product: product %}
+{% endcomment %}
+{%- unless product.has_only_default_variant -%}
+  {%- if block.settings.picker_type == 'button' -%}
+    <variant-radios
+      id="variant-radios-{{ section.id }}"
+      class="no-js-hidden"
+      data-section="{{ section.id }}"
+      data-url="{{ product.url }}"
+      {% if update_url == false %}data-update-url="false"{% endif %}
+      {{ block.shopify_attributes }}
+    >
+      {%- for option in product.options_with_values -%}
+        <fieldset class="js product-form__input">
+          <legend class="form__label">{{ option.name }}</legend>
+          {% render 'product-variant-options', product: product, option: option, block: block %}
+        </fieldset>
+      {%- endfor -%}
+      <script type="application/json">
+        {{ product.variants | json }}
+      </script>
+    </variant-radios>
+  {%- else -%}
+    <variant-selects
+      id="variant-selects-{{ section.id }}"
+      class="no-js-hidden"
+      data-section="{{ section.id }}"
+      data-url="{{ product.url }}"
+      data-update-url="{{ update_url }}"
```

Comment:
Actually we need the same thing for the select pickers. Sorry I missed that.

Reply:
oh, good catch Ken, totally slipped out of my mind :)